src/providers/aws.rs

```diff
@@ -779,7 +802,10 @@ impl RegionLauncher {
         private_key_path
             .write_all(private_key.as_bytes())
             .context("could not write private key to file")?;
-        debug!(log, "wrote keypair to file"; "filename" => private_key_path.path().display());
+        tracing::trace!(
+            filename = %private_key_path.path().display(),
+            "wrote keypair to file"
+        );
```

Comment:
I think we should keep this as a `debug!` - often when debugging a script I'll look back at the output to manually ssh to the machine (i.e., copy paste the output of this log statement to `ssh -i <foo> ubuntu@ip`)